css/templates/themes/2/page-vars.css

```diff
@@ -50,7 +50,7 @@
   --color-navbar-primary: var(--color-black);
   --color-navbar-font-primary: var(--color-white);
 
-  --color-input-primary: var(--color-font-accent);
+  --color-input-primary: var(--color-secondary);
```

Comment:
Changing the primary color will break other inputs , can you try to set the right secondary color (i have mentioned the variables in the steps)